Valerii/HW7/task2.py

```diff
@@ -0,0 +1,40 @@
+from math import pi
+
+
+def area_rectangle(a, b):
+    """ This function calculates the area of recnangle with sides:
+    a, b
+    """
+    return a*b
+
+def area_triangle(a, h):
+    """ This function calculates the area of triangle 
+    with side 'a' and hight 'h' to it.
+    """
+    return a*h/2
+
+def area_circle(r):
+    """ This function calculates the area of circle with radius 'r'."""
+    return pi*r**2
+
+def main():
+    """ The main function, that calculates the area of rectangle, triangle or circle
+    depending on the user's choise.    
+    """
+    choise = input('Hello, what area do you want to calculate?\nEnter rectange/triangle/circle: ')
```

Comment:
choise = input('Hello, what area do you want to calculate?\nEnter rectange/triangle/circle: ').lower()